packages/glimmer-runtime/lib/compiled/opcodes/vm.ts

```diff
@@ -306,6 +313,77 @@ export class EvaluateOpcode extends Opcode {
   }
 }
 
+export class EvaluatePartialOpcode extends Opcode {
+  public type = "evaluate-partial";
+  public compiler: CompileInto & SymbolLookup;
+  public name: CompiledExpression<any>;
+
+  constructor({ name, compiler }: { compiler: CompileInto & SymbolLookup, name: CompiledExpression<any> }) {
+    super();
+    this.name = name;
+    this.compiler = compiler;
```

Comment:
You already added `this.compiler.getSymbolTable()`, and the only place `this.compiler` is used in this class is to call that method, so it seems good that you just pass the symbol table from the caller instead of doing it in here.